.github/CONTRIBUTING.md

```diff
@@ -19,12 +19,12 @@ as a personal attack. (And, of course, if someone feels it is a personal attack,
 
 ## How to submit your contribution
 
-Changes should come in the form of pull requests to the master branch of the main repository. Contributors 
```

Comment:
this "master" word relates to git, and can be changed only by maintainer because it requires a change on the git repository. Until the main branch of git is renamed, this documentation cannot be changed.